browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1518,6 +1542,10 @@ L.Control.JSDialogBuilder = L.Control.extend({
 	_metricfieldControl: function(parentContainer, data, builder, customCallback) {
 		var value;
 		var controls = builder._controlHandlers['basespinfield'](parentContainer, data, builder, customCallback);
+		if (!L.Browser.chrome) {
+			controls.spinfield.onkeypress = L.bind(builder._preventNonNumericalInput, builder);
```

Comment:
can't we add this only one time in `basespinfield` which is shared in metric and formatted field?

Reply:
are we sure that basespinfield class is numeric?
what about integers, currency?